src/Data/LLVM/BitCode/Bitstream.hs

```diff
@@ -91,13 +91,15 @@ data Bitstream = Bitstream
   } deriving (Show)
 
 parseBitstream :: S.ByteString -> Either String Bitstream
-parseBitstream  = C.runGet (runGetBits getBitstream)
+parseBitstream = bimap thd3 thd3
+                 . BG.runGetOrFail (runGetBits getBitstream)
+                 . L.fromChunks . (:[])
```

Comment:
`bytestring`'s [`fromStrict`](https://hackage.haskell.org/package/bytestring-0.11.3.0/docs/Data-ByteString.html#v:fromStrict) function would be a more idiomatic way to convert from a strict to lazy `ByteString`. This function was introduced in `bytestring-0.10`, so you'd need to bump the lower version bounds in the `.cabal` file accordingly.